src/csharp/Grpc.IntegrationTesting/ClientRunners.cs

```diff
@@ -179,6 +179,9 @@ public ClientStats GetStats(bool reset)
                 statsResetCount.Increment();
             }
 
+            GrpcEnvironment.Logger.Info("[ClientRunnersImpl.GetStats] GC collection counts: gen0 {0}, gen1 {1}, gen2 {2}, gen3 {3} (histogram reset count:{4}, seconds since reset: {5})",
+                GC.CollectionCount(0), GC.CollectionCount(1), GC.CollectionCount(2), GC.CollectionCount(3), statsResetCount.Count, secondsElapsed);
```

Comment:
Good idea, we can do that if we start looking at GC stats more often and subtracting 4 numbers becomes annoying.